packages/contracts-bedrock/src/L1/DataAvailabilityChallenge.sol

```diff
@@ -66,13 +66,23 @@ contract DataAvailabilityChallenge is OwnableUpgradeable, ISemver {
     /// @notice An event that is emitted when the bond size required to initiate a challenge changes.
     event RequiredBondSizeChanged(uint256 challengeWindow);
 
+    /// @notice An event that is emitted when the percentage of the resolving cost to be refunded to the resolver changes.
+    event ResolverRefundPercentageChanged(uint256 resolverRefundPercentage);
+
     /// @notice An event that is emitted when a user's bond balance changes.
     event BalanceChanged(address account, uint256 balance);
 
     /// @notice Semantic version.
     /// @custom:semver 0.0.0
     string public constant version = "0.0.0";
 
+    /// @notice The fixed cost of resolving a challenge.
+    uint256 public constant fixedResolutionCost = 44200;
+
+    /// @notice The variable cost of resolving a callenge per byte of calldata.
+    /// @dev upper limit; 16 gas per non-zero calldata byte, 4 gas variable execution cost per byte.
+    uint256 public constant variableResolutionCost = 16 + 4;
```

Comment:
Note: these parameters might have to be tuned later. It's nontrivial to do proper statistical measurements for this purpose in forge scripts (because there isn't really a notion of cost of `calldata` in forge scripts), so we might have to add a typescript based script to find proper values for `fixedResolutionCost` and `variableResolutionCost`.